frontend/src/components/Icons/GoIcons.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <component :is="icon"></component>
+</template>
+
+<script>
+import { defineAsyncComponent, defineComponent } from 'vue'
+
+export default defineComponent({
```

Comment:
Can you register this globally in `main.ts`  so we can use icons without importing the component every time ?

Reply:
I think you could register all theses globally and since you use `defineAsyncConponent`, it would not have a performance impact.

```js
[Icons....].forEach(iconName => {
GoIcon.add (iconName)
})
app.component( "goIcon", GoIcon)
```


---

Or use `OhVue Icons`

```js
// main.js
import { createApp } from "vue";
import App from "./App.vue";
import OhVueIcon from "oh-vue-icons/dist/v3/icon.es";

import { FaFlag, RiZhihuFill } from "oh-vue-icons/icons";
OhVueIcon.add(FaFlag, RiZhihuFill);

const app = createApp(App);
app.component("v-icon", OhVueIcon);
app.mount("#app");
```

Reply:
I added a global registration for this component. Problem is that when this is used in a template my ide is still complaining about an unknown html tag. Same issue applies to `<Private>`

Reply:
You can add the tag as "Custom Html Tag". I'm sure there is a better way